services/mailservice/mail_test.go

```diff
@@ -84,7 +84,7 @@ func TestMailConnectionAdvanced(t *testing.T) {
 
```

Comment:
Sorry, but but Github's amazing review UI won't let me comment on line 66. Let's move the `require.Nil` call above `defer conn.Close()`. Otherwise, it panics if inbucket is not running.